packages/p4-analyzer-vscode/package.json

```diff
@@ -67,7 +67,7 @@
 							"warn",
 							"error"
 						],
-						"markdownDescription": "The optional `'--loglevel'` argument to supply to the P4 Analyzer server executable.\n\nUsed only in conjunction with *Absolute Server Path*."
+						"markdownDescription": "The optional `'--loglevel'` argument to supply to the P4 Analyzer server executable.\\n\nUsed only in conjunction with *Absolute Server Path*."
```

Comment:
Are these escapes by design? If so, you've missed one.

I don't think they're required for the Markdown to be rendered correctly though.

Reply:
Opps, did a find and replace and must have changed it. Will fix and merge